requirements.txt

```diff
@@ -40,4 +40,5 @@ transforms3d
 urllib3[secure]
 pytest_console_scripts
 pytest-xdist
+pytest_cases
```

Comment:
This shouldn't really be here. We should be keeping a separate list of test-only dependencies, probably like:

```
pip install spinalcordtoolbox[testing]
```

Reply:
not our problem here though.